scene/gui/popup_menu.cpp

```diff
@@ -2162,6 +2195,7 @@ void PopupMenu::_get_property_list(List<PropertyInfo> *p_list) const {
 
 void PopupMenu::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("activate_item_by_event", "event", "for_global_only"), &PopupMenu::activate_item_by_event, DEFVAL(false));
+	ClassDB::bind_method(D_METHOD("set_item_order", "id", "order"), &PopupMenu::set_item_order, DEFVAL(-1), DEFVAL(0));
```

Comment:
```suggestion
	ClassDB::bind_method(D_METHOD("set_item_order", "id", "order"), &PopupMenu::set_item_order);
```
These default values doesn't make sense, if `id` is `-1` it won't affect any item, and order should be provided if you're setting it no?